server/http/helloapp/oauth.go

```diff
@@ -50,11 +42,19 @@ func (h *helloapp) GetOAuthConfig() (*oauth2.Config, error) {
 			AuthURL:  conf.MattermostSiteURL + "/oauth/authorize",
 			TokenURL: conf.MattermostSiteURL + "/oauth/access_token",
 		},
-		RedirectURL: h.AppURL(PathOAuth2Complete),
+		// RedirectURL: h.AppURL(PathOAuth2Complete), - not needed, OAuther will configure
```

Comment:
Since we no longer use PathOauth2Complete we may consider removing the declaration of this constant.

Reply:
We use it in the manifest (CallbackURL). Maybe we shouldn't - should come from oAuther somehow.

Reply:
Understood. Yes, I think the oauther should be able to give that information. For the time being it can continue as it is.